init.lua

```diff
@@ -112,19 +114,39 @@ local KILL_OK = false
 local AREA_ACTIVATE = KILL_OK
 local AREA_NOACTIVATE = KILL_NO
 
+local savemodeToString = "FALSCH" -- only for debugging
```

Comment:
Bad use of upper/lower case ; `"FALSCH"` (string, not in English), should be primitive `false`